index.html

```diff
@@ -1,12 +1,49 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" ng-app= 'superheroesApp'>
 <head>
   <meta charset="UTF-8">
   <title>Superhero Phone Book</title>
+  <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/angularjs/1.5.6/angular.min.js"></script>
+  <script type="text/javascript" src="app.js"></script>
   <link rel="stylesheet" href="style.css">
 </head>
-<body>
+<body ng-controller='superheroesCtrl as vm'>
+
+  <div class="wrapper">
+    <div class="top">
+           <h1>Superheroes Assemble</h1>
+           <div class="marvel_logo"></div>
+           <div class="versus"></div>
+           <div class="dc_logo"></div>
+
+
+    </div>
+     search a hero: <input type="text" ng-model= "searchText">
+    <div class="superheroes" ng-repeat="superhero in vm.superheroes | filter: searchText" | ng-class = "{'marvel': superhero.comics_affiliation === 'Marvel', 'dc': superhero.comics_affiliation === 'DC' }">
+
+        <h2>{{$index + 1}}. {{superhero.character_name}} - {{superhero.name}} - {{superhero.comics_affiliation}} </h2> <br>
+
+        <button type="button" name="button" ng-click = 'view = !view'>Image</button>
+
+        <p ng-show="view"><img src={{superhero.image_url}} alt="Superhero image"></p>
+
+
+
+    </div>
+
+    <div class="new" >
+      name: <input type="text" ng-model='vm.name'>
+       character name: <input type="text" ng-model='vm.character_name'>
```

Comment:
While you can attach these new values directly to the view-model, I would recommend namespacing them in an object that clearly represents what you're trying to do. Something like `newHero`. That way you could rewrite your `ng-model` statements so they look something like `ng-model="vm.newHero.characterName`